jupyter-on-gke/iap_module/deployments/backend-config.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: cloud.google.com/v1
+kind: BackendConfig
+metadata:
+  name: config-default
+  namespace: ray
+spec:
+  iap:
+    enabled: true
+    oauthclientCredentials:
+      secretName: my-secret
```

Comment:
Where is this created?

Reply:
The deployments will be created by Terraform and the variables will be passed in my Terraform. The TF files will be in the next PR.